cmd/snap-mgmt/snap-mgmt.sh.in

```diff
@@ -112,6 +124,12 @@ purge() {
 
     echo "Removing snapd catalog cache"
     rm -f /var/cache/snapd/*
+
+    if test -d /etc/apparmor.d; then
+        echo "Removing extra snap-confine apparmor rules"
+        # shellcheck disable=SC2046
+        rm -f /etc/apparmor.d/$(echo "$SNAP_UNIT_PREFIX" | tr '-' '.').core.*.usr.lib.snapd.snap-confine
```

Comment:
You cannot guarantee the path for snap-confine is in fact `/usr/lib/snapd`. Please correct this appropriately.

Reply:
AFAIU this is the path inside the `core` snap, not your host's `$(libexecdir)/snapd/snap-confine`

Reply:
On all systems that aren't Ubuntu, they will be the host's. No one else has re-exec enabled.

Reply:
I'll double check this and get back to you.